### ISSUE TITLE
Clocks WIdget

### ISSUE THREAD

**Issue by samvthom16** [akvo/Akvo-Theme#167]
Linda: The times on the clocks are not correct. Europe is 1 hour too late (looks like it is still on summertime). 
South Asia is half an hour too early. 

Text on South East Asia & Pacific should be "Apa kabar? (So change Apu to Apa and add a question mark)

**Comment by designbychris** [comment]
@samvthom16 did you sort this or do you need me to look at it?

**Comment by samvthom16** [comment]
@designbychris Yes please. I have updated the text but not familiar with the timings. 

The file that needs to be updated is Akvo-Theme\lib\templates\clocks.php

**Comment by designbychris** [comment]
done :)
